src/_pytest/assertion/rewrite.py

```diff
@@ -273,7 +273,7 @@ def get_data(self, pathname: Union[str, bytes]) -> bytes:
         with open(pathname, "rb") as f:
             return f.read()
 
-    if sys.version_info >= (3, 9):
+    if sys.version_info >= (3, 10):
 
         def get_resource_reader(self, name: str) -> importlib.abc.TraversableResources:  # type: ignore
             from types import SimpleNamespace
```

Comment:
How about we move these two imports outside the function? This would at least caught the error in the correct Python versions.

```python
    if sys.version_info >= (3, 10):
        from types import SimpleNamespace
        from importlib.readers import FileReader

        def get_resource_reader(self, name: str) -> importlib.abc.TraversableResources:  # type: ignore
            return FileReader(SimpleNamespace(path=self._rewritten_names[name]))
```

Reply:
Hmm nevermind that, that local import does not work.